crates/re_data_store/src/entity_properties.rs

```diff
@@ -114,37 +118,36 @@ impl ExtraQueryHistory {
 
 // ----------------------------------------------------------------------------
 
-/// Get the latest value of the transform
-///
-/// We first look for the transform in the classic storage system since that's
-/// what most users are still using. If we don't find the transform there, then
-/// we check to see if it exists in the arrow storage.
-pub fn query_transform(
+/// Get the latest value for a given [`re_log_types::msg_bundle::Component`].
+pub fn query_latest<C: DeserializableComponent>(
```

Comment:
should there be something in the name or comment on how this expects that there is only a single component of the type on the entity?

Reply:
~That's already guaranteed by the data model :thinking:~ Ha, you mean a single instance. Yes.